examples/finnegans_wake_demo/README.md

```diff
@@ -11,14 +11,25 @@ without revealing private keys to intermediary entities.
 5. Bob receives and reconstructs the Enrico from Policy public key and Enrico public key
 6. Bob retrieves the original message from Enrico and MessageKit
 
+There are two version of the example, one federated example using a local network
+and another example using the nucypher application development tesnet: Lynx.
 
-### Run a fleet of federated Ursulas
+### Federated Demo
+
+First run the local federated network:
 `python ../run_demo_ursula_fleet.py`
 
+Then run the demo:
+`python finnegans-wake-demo-federated.py`
 
-### Download the Book!
-`./download_finnegans_wake.sh` 
+### Testnet Demo
 
+First, configure the demo.  Be sure tat alice's address has some Goerli ETH.
+```bash
+export DEMO_PROVIDER_URI=<GOERLI RPC ENDPOINT>
```

Comment:
Are they not defined right in the examples so that the users don't forget to set them?

Reply:
There is an example of a seednode address in the worker docs but not an ethereum endpoint.  For demo purposes I think this will most often be a remote ethereum service like Infura.